docs/pages/setup/reference/networking.mdx

```diff
@@ -50,7 +50,8 @@ following use cases:
 
 Some networks funnel all connections through a proxy server where they can be
 audited and access control rules can be applied. For these scenarios, Teleport
-supports HTTP CONNECT tunneling.
+supports HTTP CONNECT tunneling for `tsh` and for `teleport` when running in
```

Comment:
TBH I'm not sure this makes it clearer than before. Could we instead list scenarios in which HTTP_PROXY variables take effect? For example, "logging into the cluster with tsh", "reverse tunnel agents dialing back to the proxy with/without TLS routing", anything else?